contracts/deploy/initialization/FuseTypes.sol

```diff
@@ -0,0 +1,339 @@
+
+uint16 constant MORPHO_SUPPLY_WITH_CALLBACK_DATA_FUSE_ID = 46;
+string constant MORPHO_SUPPLY_WITH_CALLBACK_DATA_FUSE_NAME = "Morpho_Supply_With_Callback_Data_Fuse";
+
+uint16 constant MORPHO_CLAIM_FUSE_ID = 47;
+string constant MORPHO_CLAIM_FUSE_NAME = "Morpho_Claim_Fuse";
+
+uint16 constant PENDLE_REDEEM_PT_AFTER_MATURITY_FUSE_ID = 48;
+string constant PENDLE_REDEEM_PT_AFTER_MATURITY_FUSE_NAME = "Pendle_Redeem_PT_After_Maturity_Fuse";
+
+uint16 constant PENDLE_SWAP_PT_FUSE_ID = 49;
+string constant PENDLE_SWAP_PT_FUSE_NAME = "Pendle_Swap_PT_Fuse";
+
+uint16 constant PLASMA_VAULT_REQUEST_SHARES_FUSE_ID = 50;
+string constant PLASMA_VAULT_REQUEST_SHARES_FUSE_NAME = "Plasma_Vault_Request_Shares_Fuse";
+
+uint16 constant PLASMA_VAULT_REDEEM_FROM_REQUEST_FUSE_ID = 51;
+string constant PLASMA_VAULT_REDEEM_FROM_REQUEST_FUSE_NAME = "Plasma_Vault_Redeem_From_Request_Fuse";
+
+uint16 constant RAMSES_V2_BALANCE_FUSE_ID = 52;
+string constant RAMSES_V2_BALANCE_FUSE_NAME = "Ramses_V2_Balance_Fuse";
+
+uint16 constant RAMSES_V2_COLLECT_FUSE_ID = 53;
+string constant RAMSES_V2_COLLECT_FUSE_NAME = "Ramses_V2_Collect_Fuse";
+
+uint16 constant RAMSES_V2_MODIFY_POSITION_FUSE_ID = 54;
+string constant RAMSES_V2_MODIFY_POSITION_FUSE_NAME = "Ramses_V2_Modify_Position_Fuse";
+
+uint16 constant RAMSES_V2_NEW_POSITION_FUSE_ID = 55;
+string constant RAMSES_V2_NEW_POSITION_FUSE_NAME = "Ramses_V2_New_Position_Fuse";
+
+uint16 constant RAMSES_CLAIM_FUSE_ID = 56;
+string constant RAMSES_CLAIM_FUSE_NAME = "Ramses_Claim_Fuse";
+
+uint16 constant UNISWAP_V2_SWAP_FUSE_ID = 57;
+string constant UNISWAP_V2_SWAP_FUSE_NAME = "Uniswap_V2_Swap_Fuse";
+
+uint16 constant UNISWAP_V3_BALANCE_FUSE_ID = 58;
+string constant UNISWAP_V3_BALANCE_FUSE_NAME = "Uniswap_V3_Balance_Fuse";
+
+uint16 constant UNISWAP_V3_COLLECT_FUSE_ID = 59;
+string constant UNISWAP_V3_COLLECT_FUSE_NAME = "Uniswap_V3_Collect_Fuse";
+
+uint16 constant UNISWAP_V3_MODIFY_POSITION_FUSE_ID = 60;
+string constant UNISWAP_V3_MODIFY_POSITION_FUSE_NAME = "Uniswap_V3_Modify_Position_Fuse";
+
+uint16 constant UNISWAP_V3_NEW_POSITION_FUSE_ID = 61;
+string constant UNISWAP_V3_NEW_POSITION_FUSE_NAME = "Uniswap_V3_New_Position_Fuse";
+
+uint16 constant UNISWAP_V3_SWAP_FUSE_ID = 62;
+string constant UNISWAP_V3_SWAP_FUSE_NAME = "Uniswap_V3_Swap_Fuse";
+
+uint16 constant UNIVERSAL_TOKEN_SWAPPER_FUSE_ID = 63;
+string constant UNIVERSAL_TOKEN_SWAPPER_FUSE_NAME = "Universal_Token_Swapper_Fuse";
+
+uint16 constant UNIVERSAL_TOKEN_SWAPPER_ETH_FUSE_ID = 64;
+string constant UNIVERSAL_TOKEN_SWAPPER_ETH_FUSE_NAME = "Universal_Token_Swapper_Eth_Fuse";
+
+uint16 constant UNIVERSAL_TOKEN_SWAPPER_WITH_VERIFICATION_FUSE_ID = 65;
+string constant UNIVERSAL_TOKEN_SWAPPER_WITH_VERIFICATION_FUSE_NAME = "Universal_Token_Swapper_With_Verification_Fuse";
+
+library FuseTypes {
+    function getAllFuseId() internal pure returns (uint16[] memory) {
+        uint16[] memory fuseIds = new uint16[](65);
```

Comment:
can you order constants in this array in the same way like are done in constants declaration?